Include/cpython/object.h

```diff
@@ -469,12 +469,11 @@ passed as second argument to Py_TRASHCAN_BEGIN().
 PyAPI_FUNC(int) _PyTrash_begin(PyThreadState *tstate, PyObject *op);
 PyAPI_FUNC(void) _PyTrash_end(PyThreadState *tstate);
 
+/* Python 3.13 private API, invoked by the Py_TRASHCAN_BEGIN(). */
```

Comment:
- We should drop the version in the comment, so we don't have to change it every year.
- It's pretty clear that this is a private API as denoted by the `_Py` prefix.

```suggestion
/* Invoked by the Py_TRASHCAN_BEGIN(). */
```